src/test/_NumberToWords.java

```diff
@@ -0,0 +1,30 @@
+package test;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import convertion.NumberToWords;
+
+public class _NumberToWords {
+
+	@Test
+	public void withoutCents() {
+		String res=NumberToWords.convert(23);
```

Comment:
Don’t make unnecessary assertions 
Which specific behaviour are you testing? It’s counterproductive to Assert() anything that’s also asserted by another test: it just increases the frequency of pointless failures without improving unit test coverage one bit. This also applies to unnecessary Verify() calls – if it isn’t the core behaviour under test, then stop making observations about it! Sometimes, TDD folks express this by saying “have only one logical assertion per test”. 
Remember, unit tests are a design specification of how a certain behaviour should work, not a list of observations of everything the code happens to do.